dash_docs/chapters/dash_enterprise/dash_enterprise_chapters.py

```diff
@@ -3241,98 +3461,205 @@ def snapshot_page(n_clicks, url, wait_selector):
+        Only the app owner and admin users can deploy to the application.\
 
-        rc.Markdown(
-            '''
-            These applications require using a `worker`
-            process. When using a `worker` process in your `Procfile`,
-            you will have to explicitly start it after deploying. To
-            scale a `worker` process, provide a `DOKKU_SCALE` file with
-            something like this:
-            '''),
+        1. Only the app owner and admin users can deploy to the application. 
+        If you are given admin rights you will be able to deploy changes to the app as well.\
 
-        rc.Markdown('```\n$web=1\nworker=1```',
-                              style=styles.code_container),
+        2. Pushing via ssh deploy method](https://dash.plotly.com/dash-enterprise/ssh) after adding your ssh key to the app owner's account. 
+        For this method to work, you must create a separate ssh key specifically for the other users account. 
+        If you upload the same key to two different users, 
+        then Dash Enterprise will not know which user is deploying and will complain and say you don't have permission.\
+
+        3. Pushing the master branch from the repository with the app owner's credentials after setting up a CI pipeline working in a repository.\
+        '''
+    ),
+
+    rc.Markdown(
+        '''
+        ### Please make sure you have the correct access rights and the repository exists
+        '''
+    ),
+
+    rc.Markdown(
+        '''
+        ```shell
+        $ git push plotly master
+        [...]
+        dokku@<dash-enterprise-server>: Permission denied (publickey).
+
+        fatal: Could not read from remote repository.
+
+        Please make sure you have the correct access rights
+
+        and the repository exists.
+        ```        
+        '''
+    ),
+
+    rc.Markdown(
+        '''
+        This can happen when your `~/.ssh/config` is improproperly configured. See  <dccLink href="/dash-enterprise/ssh" children="ssh"/> 
+        page for more details.
+        '''
+    ),
+
+    rc.Markdown(
+        '''
+        ### remote: Access denied
+        '''
+    ),
+
+    rc.Markdown(
+        '''
+        https:
+
+        ```shell
+        $ git push plotly master
+        [...]
+        Counting objects: 3, done.
+
+        Delta compression using up to 8 threads.
+
+        Compressing objects: 100% (3/3), done.
+
+        Writing objects: 100% (3/3), 5.18 KiB | 2.59 MiB/s, done.
+
+        Total 3 (delta 0), reused 0 (delta 0)
+
+        remote: User <your-user-name> does not have permissions to run git-hook on <your-dash-app>, or <your-dash-app> does not exist
+
+        remote: Access denied
+
+        To <your-dash-enterprise-host>/GIT/<your-dash-app>
+
+        ! [remote rejected] master -> master (pre-receive hook declined)
+
+        error: failed to push some refs to '<your-dash-enterprise-host>:<your-dash-app>'
+        '''
```

Comment:
```suggestion
        '''
        ```suggestion
        If deploying with SSH, then this can happen when your `~/.ssh/config` is improperly configured. See  <dccLink href="/dash-enterprise/ssh" children="ssh"/> for more details.
        
        If deploying with HTTPS, then this can happen if your username or password is incorrect. Note that usernames and passwords in Dash Enterprise are case sensitive: the username `Chris.Parmer` is different than the username `chris.parmer`.
```
```